truffle.js

```diff
@@ -1,16 +1,12 @@
-var DefaultBuilder = require("truffle-default-builder");
-
 module.exports = {
-    build: new DefaultBuilder({}),
-
     networks: {
         "development": {
-            host: "testrpc",
+            host: "172.19.0.2",
             port: 8545,
             network_id: "*"
         },
         "testnet": {
-            host: "parity-testnet",
+            host: "52.221.123.6",
```

Comment:
Assuming these got changed by accident? I don't think those IPs are static at all, or portable. did you have trouble with the hostnames?